setup.py

```diff
@@ -42,6 +42,9 @@
           'setuptools',
           'plone.rest >= 1.0a6',  # json renderer moved to plone.restapi
           'PyJWT',
+          'five.intid >= 1.1.2',
```

Comment:
@erral could you please elaborate why we need at least 1.1.2 (and possibly add a note to setup.py). Those requirements usually lead to problems at some point, so I try to avoid them whenever possible.